DEVELOPER.rst

```diff
@@ -1,71 +1,148 @@
-=================
- Install package
-=================
+Developer guide
+###############
 
-Install in develop mode to notice changes immediately.
+This document should guide you through the setup of your local developer
+environment, as well as through the release process of a new version.
 
-.. code::
+====================================
+Setting up a development environment
+====================================
 
-   python setup.py develop
+Download code
+-------------
 
-=================
- Doing a Release
-=================
+First clone the repository to your local machine::
 
-Update version
---------------
+    $ git clone https://github.com/bio-phys/MDBenchmark.git
 
-Change version info in `mdbenchmark/__init__.py`
+Create a virtual environment
+----------------------------
 
-Updating CHANGELOG
-------------------
+If you are using ``conda``, you can easily create a `virtual environment`_::
```

Comment:
this is called a conda environment. virtual environments are something different in python lingo